src/file/mod.rs

```diff
@@ -467,17 +467,19 @@ impl AsRef<OsStr> for TempPath {
 /// # Resource Leaking
 ///
 /// If the program exits before the `NamedTempFile` destructor is
-/// run, such as via [`std::process::exit()`], by segfaulting, or by
-/// receiving a signal like `SIGINT`, then the temporary file
-/// will not be deleted.
+/// run then the temporary file will not be deleted. This can happen
```

Comment:
```suggestion
/// run, the temporary file will not be deleted. This can happen
```

(Reads better? There may be some English rule here, that wasn't my best class)

Reply:
English is complicated: https://www.dailywritingtips.com/comma-before-but/

Since the implied subject is "the temporary file" then a single "then" may be used without a comma.
I'm about 70% sure I'm correct on this.

Reply:
It's a separate clause.

https://www.grammarly.com/blog/conditional-sentences/

The "then" is optional and usually omitted, but the comma isn't.

Reply:
Fixed.